.travis.yml

```diff
@@ -126,6 +126,7 @@ matrix:
           name: Python 3.7 with all optional dependencies
           env: PYTHON_VERSION=3.7
                TOXENV='py37-test-alldeps'
+               TOXPOSARGS="--remote-data=any"
```

Comment:
Since there are already some jobs with remote data access, I think this is not needed? (there is a job that is an allowed failure which has remote data access)

Reply:
Having the a job with remote data on Windows would be useful as it will trigger exactly this type of bug with certificates (maybe also some other Windows firewall/service weirdness).
But maybe it should be separate from this final job?

Reply:
One could do a cron job with remote data on windows; it doesn't seem something that is needed to test PRs, but would be good to check.

Reply:
Agreed, is there something you have setup for this kind of periodic test?

Reply:
Re: cron -- Yes, you can set it up as such in `.travis.yml`. For example:

https://github.com/astropy/astropy/blob/bad46e22d293ca1c891bef0d0fae9e78c324eec5/.travis.yml#L81

It runs nightly.